errands-routes.go

```diff
@@ -1,31 +1,34 @@
+//nolint:golint // TODO
 package main
 
 import (
+	"encoding/json"
+	"errors"
 	"io"
-	// "fmt"
+	"net/http"
 	"sort"
 
 	log "github.com/sirupsen/logrus"
-	// "time"
-	"encoding/json"
-	"errors"
-	"net/http"
 
 	gin "github.com/gin-gonic/gin"
 	schemas "github.com/polygon-io/errands-server/schemas"
 	utils "github.com/polygon-io/errands-server/utils"
 )
 
+//nolint:gocognit // TODO
```

Comment:
this is insane, this code is clearly fine.

Reply:
It's not great.

Reply:
I'll remove this now that our threshold is higher.